src/App.js

```diff
@@ -93,16 +97,40 @@ export default function App() {
    */
   const getQuestionsByRow = (rowIndex, triviaData) => {
     const rowData = [];
+    const emptyState = {
+      points: 0,
+      answer: 'Empty',
+      question: 'Empty',
+    };
     triviaData.forEach((category, categoryIndex) => {
-      rowData.push(
-        <Col className='q-col align-self-center'>
-          <QuestionCard
-            height={height}
-            questionDetails={category.questions[rowIndex]}
+      /* If no question is given in row fill Card with Empty State else fill
+      it with question info */
+
+      if (category.questions[rowIndex] === undefined) {
+        rowData.push(
+          <Col
             key={categoryIndex + '-' + rowIndex}
-          ></QuestionCard>
-        </Col>,
-      );
+            className='q-col align-self-center'
+          >
+            <QuestionCard
+              height={height}
+              questionDetails={emptyState}
```

Comment:
the block duplication here can be simplified with `questionDetails={category.questions[rowIndex] || emptyState}`

Reply:
Agree with @michaeladda, we can avoid the duplication by adding this check there. Thanks.